blog/_posts/2021-07-09-FastAI-tabular-data.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Working with Tabular Data in FastAI.jl
+author: Manikya Bardhan
+layout: blog
+---
+
+[FastAI.jl](https://github.com/FluxML/FastAI.jl) is a package inspired by [fastai](https://github.com/fastai/fastai), and it's goal is to easily enable creating state-of-the-art models.
+
+This blog post shows how to get started on working with tabular data using FastAI.jl and related packages. The work being presented here was done as a part of [GSoC'21](https://summerofcode.withgoogle.com/projects/#5088642453733376) under the mentorship of Kyle Daruwalla, Brian Chen and Lorenz Ohly.
```

Comment:
I think you should not undersell your work here, I am truthfully unfamiliar with the deep technical detail but saying something like "Before my GSoC project, we could only do x and y. Now we can do XY & Z together with this unified interface". This will make it very clear why someone should read this post.

Reply:
Agreed, this project was no small feat. Just look at how long it's taken other frameworks to (not) add support for new modalities!

Reply:
Thanks for the comments! I'll add this in as well.